packages/ui/src/components/Dropdown.svelte

```diff
@@ -0,0 +1,199 @@
+<!--
+// Copyright © 2021 Anticrm Platform Contributors.
+// 
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import { IntlString } from '@anticrm/status'
+
+  import ui from '../component'
+  import type { DropdownItem } from '../types'
+
+  import ArrowUp from './icons/Up.svelte'
+  import ArrowDown from './icons/Down.svelte'
+  import Label from './Label.svelte'
+
+  export let items: DropdownItem[]
+  export let selected: DropdownItem['id'] | undefined
+  export let title: IntlString | undefined
+
+  let isDisabled = false
+  $: isDisabled = items.length === 0
+
+  let isOpened = false
+  let selectedItem = items.find((x) => x.id === selected)
+  $: selectedItem = items.find((x) => x.id === selected)
+  $: if (selected === undefined && items[0] !== undefined) {
+    selected = items[0].id
+  }
+
+  function onItemClick (id: DropdownItem['id']) {
+    selected = id
+  }
+
+  function onClick () {
+    isOpened = !isOpened
+  }
+
+  function onClickOutside () {
+    isOpened = false
+  }
+
+  function clickOutside (node: any, onEventFunction: any) {
+    const handleClick = (event: any) => {
+      const path = event.composedPath()
+
+      if (!path.includes(node)) {
+        onEventFunction()
+      }
+    }
+
+    document.addEventListener('click', handleClick)
```

Comment:
Is this is only possible variant? Could <svelte:window> be used?

https://svelte.dev/tutorial/svelte-window